Source/iOS/OIDAuthState+IOS.m

```diff
@@ -18,17 +18,23 @@
 
 #import "OIDAuthState+IOS.h"
 
-#import "OIDExternalUserAgentIOS.h"
+#if TARGET_OS_MACCATALYST
```

Comment:
I need to learn more about Mac Catalyst. Instead of OIDAuthState+IOS supporting both iOS and Catalyst, can we instead have a OIDAuthState+MacCatalyst  that the Catalyst target imports exclusively?  Or do both build from the same target?

Reply:
Catalyst builds from the same target as the iOS framework. I don't think we can have a separate target for Catalyst (maybe I'm wrong?) or at least I don't see how that's possible.

Reply:
Understood, thanks for clarifying.